scripts/bash/run_enceladus.sh

```diff
@@ -482,11 +482,9 @@ if [[ -z "$DRY_RUN" ]]; then
     STATE='NOT FINISHED'
 
     while [[ "$STATE" != "FINISHED" && "$STATE" != "FAILED" && "$STATE" != "KILLED" ]];  do
+      sleep 30
```

Comment:
Because now users need to wait for 30s before 1st message, how about adding `echo State: Application Starting` or something? On line 483?